st3/lsp_utils/node_distribution.py

```diff
@@ -14,7 +14,7 @@
 
 __all__ = ['NodeDistribution', 'NodeDistributionPATH', 'NodeDistributionLocal']
 
-NODE_VERSION = '14.15.4'
+NODE_VERSION = '12.20.1'
```

Comment:
`12.20.2` got released just days ago.
https://nodejs.org/dist/v12.20.2/

Reply:
Thanks, will update.

I'm thinking that there probably won't be any benefits from being always on the latest version so would not necessarily always bump the version in sync with node releases. Otherwise would likely have to figure out some solution for cleaning out the old versions.